src/helper.h

```diff
@@ -93,6 +93,10 @@ class helper: public RefCountable
     /// \param needsNewServers true if new servers must started, false otherwise
     void handleKilledServer(HelperServerBase *srv, bool &needsNewServers);
 
+    /// Reacts to an insufficient number of servers after a server death
+    /// \param madeProgress whether the died server responded to any requests
+    void handleLackOfServers(bool madeProgress);
```

Comment:
To sync with the other requests, to explain the difference with handleKilledServer(), and to improve style:

```suggestion
    /// Reacts to unexpected server death(s), including a failure to start server(s)
    /// and an unexpected exit of a previously started server. \sa handleKilledServer()
    /// \param madeProgress whether the died server responded to any requests
    void handleFewerServers(bool madeProgress);
```